src/components/Gradebook/SearchControls.test.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { SearchControls } from './SearchControls';
```

Comment:
In ben's PR he also tested mapDispatchToProps and mapStateToProps, so I think we should as well